sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -115,9 +115,15 @@ internal class AmqpConnectionScope : TransportConnectionScope
         private FaultTolerantAmqpObject<AmqpConnection> ActiveConnection { get; }
 
         /// <summary>
-        ///  The controller responsible for managing transactions.
+        ///  The controller responsible for managing transactions. This is not used
```

Comment:
I understand what this means, but I find it counter-intuitive.   The transaction controller isn't used when the transaction group is set.... it may be helpful to add a line about why,  or tweak the name to something like `LocalTransactionController` or `UngroupedTransactionController`

Reply:
... or `SingleEntityTransactionController`  ?